Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -82,7 +82,7 @@ protected override void OnEnable()
                 }
             }
 
-            if (!MixedRealityToolkit.ConfirmInitialized())
+            if (!MixedRealityToolkit.IsInitialized)
```

Comment:
`ConfirmInitialized()` is intentionally called here to ensure that the setup phase works correctly. What's the benefit of just bailing if it's not initialized?

Reply:
These two ConfirmInitialized calls - and only these calls - were causing game objects (camera / play space transform) to be created during OnDestroy in play mode, which Unity really doesn't like.

Bailing in the inspector until the toolkit is initialized elsewhere seems to work. The toolkit still gets initialized in the editor without causing life cycle problems. And at runtime it's obviously not an issue.